src/docket/docket.py

```diff
@@ -192,11 +204,18 @@ def __init__(
         self.execution_ttl = execution_ttl
         self.enable_internal_instrumentation = enable_internal_instrumentation
         self._cancel_task_script = None
+        self._cluster_client = None
 
         self.result_storage: AsyncKeyValue
         if url.startswith("memory://"):
             self.result_storage = MemoryStore()
+        elif is_cluster_url(url):
+            # key_value's RedisStore doesn't support Redis Cluster yet
+            # Use MemoryStore as a temporary workaround
+            # TODO: Add cluster-aware result storage when key_value supports it
+            self.result_storage = MemoryStore()
```

Comment:
Why couldn't `RedisStore` take the `{{{name}}}:results` key prefix format too?  Is it because py-key-value needs to use a cluster client as well?